tests/ui/test_odf_topology.py

```diff
@@ -98,16 +96,19 @@ def test_validate_topology_configuration(
         """
 
         topology_tab = PageNavigator().nav_odf_default_page().nav_topology_tab()
+        random_node_name = random.choice(get_worker_nodes())
+
+        if config.DEPLOYMENT["external_mode"]:
+            if topology_tab.is_entity_present(random_node_name):
```

Comment:
Can you use `and`?
```
if config.DEPLOYMENT["external_mode"] and topology_tab.is_entity_present(random_node_name):

```

Reply:
sure, fixing

Reply:
removed this block.
I rather not add this change to this PR.
Thanks